projects/pgai/db/tests/vectorizer/test_vectorizer.py

```diff
@@ -201,7 +201,9 @@ def psql_cmd(cmd: str) -> str:
     return str(proc.stdout).strip()
 
 
-@pytest.mark.skipif(os.getenv("PG_MAJOR") == "15", reason="does not support pg15")
```

Comment:
I don't understand these changes, because the format that's red is not even what's in `main` so 🤷🏼

Reply:
you need to rebase your change against main